### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by tasleson**
Some things that popped out when looking around in my editor and from running pylint on the source base.  Please scrutinize before picking any commits up as I haven't tested any of this, but I think they are pretty straight forward fixes.

### CODE REVIEW

**Comment by bmr-cymru** [comment]
Thanks - most of these look reasonable, but please stick to the convention you see in master for subject formatting: we use the Python module name where we're changing source, e.g. "boom.bootloader", or a string like "doc" or "build" for changes elsewhere. We don't use a `.py` suffix in the subject.

Feel free to force push an updated branch or I'll correct that as I go through and merge.

I tend to use `pycodestyle` (the successor to `pep8`) for style checking in my projects now - we ditched `pylint` for sos a number of years ago because we weren't very happy with some of its rules. I have a backlog of style fixes right now however.

**Comment by tasleson** [comment]
@bmr-cymru Subjects updated as requested, thanks.

**Comment by bmr-cymru** [comment]
Thanks @tasleson, appreaciate it!

**Comment by bmr-cymru** [comment]
It looks like you based your branch on a tree from the 29th March:

```
$ git merge-base master tasleson/misc_fixes
6ff3e0869b9818f4cc477e5771bc9e72352e7737
$ git show 6ff3e0869b9818f4cc477e5771bc9e72352e7737
commit 6ff3e0869b9818f4cc477e5771bc9e72352e7737 (tasleson/master, tasleson/bmr-grub2-env-vars, origin/master, origin/bmr-grub2-env-vars, bmr-grub2-env-vars)
Author: Bryn M. Reeves <bmr@redhat.com>
Date:   Fri Mar 29 11:52:44 2019 +0000

    doc: add CentOS to the list of BLS supporting distros
    
    Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
```

Happily, in this case, there are no conflicts, so I'll just rebase again and merge - but it's generally a good idea to issue a pull before setting out on work like this (esp. when the changes are small and spread around, since that increases the likelihood for conflict).

**Comment by bmr-cymru** [comment]
Ugh. Nevermind - I had three unpushed commits locally.

File: boom/command.py [hunk 1]
```diff
@@ -1462,7 +1470,7 @@ def show_legacy(selection=None, loader=BOOM_LOADER_GRUB1):
 
         :param selection: A Selection object giving selection criteria
                           for the operation
-        :param fmt: The name of a legacy boot loader format
+        :param loader: Which boot loader to use
```
Comment (bmr-cymru) [inline boom/command.py]: The rest of the patch is fine, but why obscure the fact that this is a *name*? "Which boot loader to use" doesn't give the reader much idea what to pass here... (it's a string, containing the name of a legacy format), so now someone has to go and look at the definition as well.

I'd be fine with making it consistent with the language used in `boom.legacy` itself for identical parameters:

```
    :param loader: the legacy bootloader format to operate on
```

It still requires the reader to know that legacy bootloader formats are identified by name in `boom.legacy`, but that's fairly clear and keeps the language consistent around the tree.

Comment (tasleson) [reply boom/command.py]: Leveraged your suggestion of using

`:param loader: the legacy bootloader format to operate on`\

Updated patch set, thanks.

**Comment by tasleson** [comment]
> It's actually like that for a reason - it's not something I especially liked, and it's not really something I want to export to the world from `boom`. Really, it ought to have a FIXME explaining this.
> 
> I think rather than advertise it as a first class API routine, I'd be happier moving it into the one public boom submodule that uses it (`boom.command`), and re-defining it or doing a quirky import in the test suite - the need to call it from the test suite is the reason it ended up there in the first place.

We can remove this commit from PR if you like, or I can try to share it internally without making it public.

**Comment by tasleson** [comment]
@bmr-cymru I took out the commit for renaming`_get_machine_id` to `get_machine_id`

**Pull request closed without merging**